sql/policy_api.sql

```diff
@@ -59,10 +59,31 @@ RETURNS INTEGER
 AS '@MODULE_PATHNAME@', 'ts_policy_compression_add'
 LANGUAGE C VOLATILE; -- not strict because we need to set different default values for schedule_interval
 
+CREATE OR REPLACE FUNCTION @extschema@.add_columnstore_policy(
```

Comment:
Can't we make the policy things procedures as well? It would make things more uniform for the new API

Reply:
Done for both functions.